__run_eee.sh

```diff
@@ -195,12 +195,12 @@ while [[ "${finished}" = false ]] ; do
     eefile='eee_results.dat'
     parafile_M=$( \ls parameter_sets_1_*_M.dat | grep -v scaled )
     parafile_v=$( \ls parameter_sets_1_*_v.dat )
-    python ${isdir}/codes/3_derive_elementary_effects.py -i ${model_outputs} -k ${modeloutputkey} -d ${maskfile} -m ${parafile_M} -v ${parafile_v}  -o ${eefile}
+    python3 "${isdir}"/codes/3_derive_elementary_effects.py -i ${model_outputs} -k ${modeloutputkey} -d "${maskfile}" -m "${parafile_M}" -v "${parafile_v}"  -o "${eefile}"
```

Comment:
could you please revert "python3" back to "python". I can show you how to tell your system to use "python" as default. Using "python3" here would be quite restrictive to users.